examples/custom_signaller/src/main.rs

```diff
@@ -0,0 +1,139 @@
+use custom_signaller::IrohGossipSignallerBuilder;
+use futures::{select, FutureExt};
+use futures_timer::Delay;
+use matchbox_socket::{PeerState, WebRtcSocket};
+use std::{sync::Arc, time::Duration};
+use tracing::info;
+
+const CHANNEL_ID: usize = 0;
+const LOG_FILTER: &str = "info,custom_signaller=info,iroh=error";
+
+#[cfg(target_arch = "wasm32")]
+fn main() {
+    // see  https://github.com/DioxusLabs/dioxus/issues/3774#issuecomment-2733307383
```

Comment:
I didn't look into what the issue was here, but it is now closed, does this mean we can fix this?